apps/namadillo/src/App/Staking/IncrementBonding.tsx

```diff
@@ -80,10 +80,6 @@ const IncrementBonding = (): JSX.Element => {
         </>
       ),
     }),
-    parseErrorTxNotification: () => ({
```

Comment:
If we remove this, the notification failed doesn't appear. I tested this trying to stake with a custom small amoutn of gas (0.01 NAM) and you can observe the notification there forever

![Screenshot 2025-01-31 at 14 06 47](https://github.com/user-attachments/assets/89bfeac3-0945-4d8a-b5cd-cac89d6b1646)